corehq/form_processor/backends/couch.py

```diff
@@ -0,0 +1,61 @@
+from couchdbkit import ResourceNotFound
```

Comment:
we also may want to rename this file to `backends/couch/xform.py` if we feel like there is going to be a 1-to-1 mapping from interface functions to backend functions (that's how it currently is right now). if it's not going to be like that then we can just leave as is. this'll probably be something we figure out later down the road -- just flagging for now